### PR TITLE
The fuzz tests were added for the tensorflow::core::function::Runtime class

### PR DESCRIPTION
**Pull request by OleksiiZaderykhinCapgemini**
The fuzz tests were added for the tensorflow::core::function::Runtime class.

### CODE REVIEW

File: tensorflow/security/fuzzing/core/function/runtime_client_fuzz.cc [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+  return fuzztest::InRange(1, 7);
+}
+
+auto FunctionDefDomain() {
+  return fuzztest::Map(EmptyFunctionDefGenerator, NumberOfInputArguments(), NumberOfOutputArguments());
+}
+
+void CreateFunctionFuzz(FunctionDef def) {
+  auto ctx = LocalEagerCtx();
+  Runtime rt(*ctx);
```
Comment (Konstantin-Beluchenko) [inline tensorflow/security/fuzzing/core/function/runtime_client_fuzz.cc:91]: is Runtime creation time-consuming operation?
if so maybe there is sense to move it to the test fixture?

Comment (OleksiiZaderykhinCapgemini) [reply tensorflow/security/fuzzing/core/function/runtime_client_fuzz.cc:91]: The context here holds the current state. That state is denote if the function we create is present in the context or isn't. So the creation of the context is part of the test, and so is the creation of the Runtime.

**Review by fcoUnda — CHANGES_REQUESTED** [review]
Please use the macro `tf_cc_fuzz_test` found in `tensorflow/security/fuzzing/tf_fuzzing.bzl` instead of tf_cc_shared_test. This macro automatically includes "@com_google_fuzztest//fuzztest" and "@com_google_fuzztest//fuzztest:fuzztest_gtest_main", so you need to remove them from the deps list. You can also remove "@com_google_googletest//:gtest_main", since it is not needed.

**Comment by OleksiiZaderykhinCapgemini** [comment]
> Please use the macro `tf_cc_fuzz_test` found in `tensorflow/security/fuzzing/tf_fuzzing.bzl` instead of tf_cc_shared_test. This macro automatically includes "@com_google_fuzztest//fuzztest" and "@com_google_fuzztest//fuzztest:fuzztest_gtest_main", so you need to remove them from the deps list. You can also remove "@com_google_googletest//:gtest_main", since it is not needed.

@fcoUnda, Done

**Comment by gbaned** [comment]
Hi @fcoUnda Can you please review this PR ? Thank you!

**Comment by gbaned** [comment]
Hi @fcoUnda Can you please review this PR ? Thank you!

**Comment by gbaned** [comment]
Hi @fcoUnda Can you please review this PR ? Thank you!

**Pull request closed without merging**